lib/paseto/local.rb

```diff
@@ -1,62 +1,69 @@
 module Paseto
-  class Local
-    HEADER = 'v2.local'
-    NONCE_BYTES = RbNaCl::AEAD::ChaCha20Poly1305IETF.nonce_bytes
+  module V2
+    module Local
+      HEADER = 'v2.local'
+      NONCE_BYTES = RbNaCl::AEAD::ChaCha20Poly1305IETF.nonce_bytes
 
-    NonceError = Class.new(Paseto::Error)
-    AuthError = Class.new(Paseto::Error)
-    BadMessageError = Class.new(Paseto::Error)
+      NonceError = Class.new(Paseto::Error)
 
-    def self.generate_aead_key
-      RbNaCl::Random.random_bytes(RbNaCl::AEAD::ChaCha20Poly1305IETF.key_bytes)
-    end
+      class Key
+        def self.generate
+          new(RbNaCl::Random.random_bytes(RbNaCl::AEAD::ChaCha20Poly1305IETF.key_bytes))
+        end
 
-    def self.generate_nonce
-      RbNaCl::Random.random_bytes(NONCE_BYTES)
-    end
+        def self.decode64(encoded_key)
```

Comment:
So I think this is a little bit of name collision for `Pasteo::V2::Local.decode64` vs `Pasteo.decode64`. I think it sides steps the possible confusion by keeping the names distinct.